docs/tutorials/dynamo.md

```diff
@@ -132,14 +103,31 @@ This flag can be set in `_guestDBs` collection for the corresponding geofabric.
 
 ```bash
    aws dynamodb batch-write-item \
-     --request-items '{"Music": [{"PutRequest": {"Item": {"Artist": {"S": "TestName1"},"SongTitle": {"S": "The Best1"}}}},{"PutRequest": {"Item": {"Artist": {"S": "TestName2"},"SongTitle": {"S": "The Best2"}}}}]}' \
+     --request-items '{"Music": [{"PutRequest": {"Item": {"Artist": {"S": "TestName1"},"SongTitle": {"S": "The Best1"}}}},{"PutRequest": {"Item": {"Artist": {"S": 
+"TestName2"},"SongTitle": {"S": "The Best2"}}}}]}' \
      --return-consumed-capacity INDEXES \
      --return-item-collection-metrics SIZE \
      --endpoint-url https://api-gdn.paas.macrometa.io/_api/dynamo
 ```
 
 ### Scan
 
+**Retrieve All Items:**
```

Comment:
I suggest changing the bold items in this section to h4s and removing the colons.

Reply:
Done